src/utils.ts

```diff
@@ -791,9 +837,9 @@ export function equals(a, b) {
     var length, i, keys;
     if (Array.isArray(a)) {
       length = a.length;
-      if (length != b.length) return false;
-      for (i = length; i-- !== 0;)
-        if (!equals(a[i], b[i])) return false;
+      if (a.length != b.length) return false;
+      for (i = a.length; i-- !== 0;)
+        if (!b.includes(a[i])) return false;
```

Comment:
this change is not equivalent to the previous version as we loose index equality. Is this wanted ?

Reply:
Went back to previous code